pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/FieldSettings.java

```diff
@@ -106,6 +115,10 @@ public static Map<String, Collection<FieldSettings>> getFieldSettings(@NonNull S
                             fieldSettingsList.put(type, subList);
                         }
                     }
+                } catch (JsonMappingException e) {
+                    logger.error("Unable to get the FieldSetting ", e);
+                } catch (JsonProcessingException e) {
+                    logger.error("Unable to parse the FieldSetting json ", e);
```

Comment:
please add the field_settings_id or column_name here as well

Reply:
@SimoneMaiwald fixed! Thank you